draft-ietf-rats-architecture.md

```diff
@@ -133,35 +133,35 @@ This document uses the following terms.
 
 Appraisal Policy for Evidence:
 
-: A set of rules that informs how a Verifier evaluates the validity of information about an Attester. Compare /security policy/ in {{RFC4949}}
+: A set of rules, reference values and such that is used by a Verifier to evaluate the Evidence from an Attester. Compare /security policy/ in {{RFC4949}}
```

Comment:
I liked the previous phrasing better. E.g. +1 "informs", -1 "and such"

Reply:
"A set of rules that informs a Verifier regarding the evaluation of the Evidence." or "A set of rules that informs a Verifier regarding the appraisal of the Evidence."

Reply:
Note use of 'appraisal' in section below on Attestation. Should use common wording if we mean similar things.

Reply:
Ned's wording plus mention of reference values: "A set of rules and optional reference values that informs a Verifier regarding the appraisal of the Evidence." Mention of reference values will greatly help the reader make sense quicker IMO.

Reply:
```suggestion
: A set of rules that informs a Verifier regarding the appraisal of the Evidence, e.g., "byte-compare in little-endian Claims with reference values in big-endian". Compare /security policy/ in {{RFC4949}}.
```